scripts/ZoneMinder/lib/ZoneMinder/Control/Dahua.pm

```diff
@@ -39,16 +39,26 @@ sub AUTOLOAD
     my $class = ref($self) || croak( "$self not object" );
     my $name = $AUTOLOAD;
     $name =~ s/.*://;
+    ## This seems odd... if the method existed would we even be here?
+    ## https://perldoc.perl.org/perlsub.html#Autoloading
```

Comment:
This is a hash lookup, not a function call lookup.  So basically if we have the value stored in us, return it.